src/shared/directives/conditional-show/conditional-show.directive.ts

```diff
@@ -41,7 +42,8 @@ export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
       this.condition = new ShowCondition(this.caseField.show_condition);
       // console.log('FIELD: ' + this.caseField.id + ' init. Show condition: ' + this.caseField.show_condition);
       this.formGroup = this.formGroup || new FormGroup({});
-      this.formField = this.formGroup.get(this.caseField.id);
+      this.formSubGroup = this.formSubGroup || new FormGroup({});
+      this.formField = this.formSubGroup.get(this.caseField.id) || this.formGroup.get(this.caseField.id);
```

Comment:
please tale a closer look at this line 🙂. I lookup in the **formSubGroup** first and then **formGroup** as the case field id may be same for a root case field or a complex subfield.